build/Microsoft.DotNet.Cli.DependencyVersions.props

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <CLI_MSBuild_Version>15.1.0-preview-000458-02</CLI_MSBuild_Version>
+    <CLI_MSBuild_Version>15.1.0-preview-000503-01</CLI_MSBuild_Version>
     <CLI_NETSDK_Version>1.0.0-alpha-20161230-1</CLI_NETSDK_Version>
+    <CLI_Roslyn_Version>2.0.0-rc3-61212-03</CLI_Roslyn_Version>
     <CLI_WEBSDK_Version>1.0.0-alpha-20161205-1-154</CLI_WEBSDK_Version>
```

Comment:
Had to bump Roslyn so dependencies of it + MSBuild didn't clash. This is the maximum Roslyn version CLI can take until we move to NetCore.App 1.1+